source/uax31.tex

```diff
@@ -49,8 +49,8 @@
 \rSec2[uaxid.def.rfmt]{R1a Restricted format characters}
 
 \pnum
-If an implementation of UAX \#31 wishes to allow format characters
-such as ZERO WIDTH JOINER or ZERO WIDTH NON-JOINER
+If an implementation of \UAX{31} wishes to allow format characters
+such as \uname{zero width joiner} or \uname{zero width non-joiner}
```

Comment:
Please use the full U+nnnn NAME style here, as we do everywhere else we refer to a Unicode character.

Reply:
Done!